bin/restic_backup.sh

```diff
@@ -101,3 +103,31 @@ wait $!
 #wait $!
 
 echo "Backup & cleaning is done."
+
+#
+# (optionally) Notify about backup summary stats.
+#
+# How to perform the notification is up to the user; the script only writes the info to the user-owned file in a fire
+# and forget fashion.
+#
+# One option to trigger desktop notifications on user-side is using a special FIFO file (a.k.a. pipe file), which will
+# work as a queue; plus, a user process to read from that queue and run a desktop notification command.
+#
+# TODO Clean/or rephrase the example comments below
```

Comment:
I left this in the draft PR with the TODO to clarify and show how it would work, hehe. I'll clean this in the next commit when I move the PR to final, now that the idea is clear :)